app/src/processing/app/contrib/ContributionManagerDialog.java

```diff
@@ -410,36 +404,35 @@ public void actionPerformed(ActionEvent actionEvent) {
       public void keyPressed(KeyEvent e) {
         //System.out.println(e);
         KeyStroke wc = Toolkit.WINDOW_CLOSE_KEYSTROKE;
-        if ((e.getKeyCode() == KeyEvent.VK_ESCAPE) ||
-            (KeyStroke.getKeyStrokeForEvent(e).equals(wc))) {
+        if ((e.getKeyCode() == KeyEvent.VK_ESCAPE)
```

Comment:
Some of the changes will still be subjective so a formatter isn't going to fix everything.